pom.xml

```diff
@@ -270,17 +270,17 @@
     <implementation.build>${scmBranch}@r${buildNumber}</implementation.build>
     <implementation.build.tstamp>${maven.build.timestamp}</implementation.build.tstamp>
 
-    <httpclient.version>4.5.7</httpclient.version>
-    <httpcore.version>4.4.11</httpcore.version>
+    <httpclient.version>4.5.10</httpclient.version>
+    <httpcore.version>4.4.12</httpcore.version>
     <owlapi.version>5.1.11</owlapi.version>
-    <poi.version>4.0.1</poi.version>
-    <rdf4j.version>2.4.4</rdf4j.version>
+    <poi.version>4.1.0</poi.version>
+    <rdf4j.version>3.0.0</rdf4j.version>
```

Comment:
Upgrading rdf4j is what caused your bug. Can we hold off on that? Upgrading rdf4j is worthy of being solved in its own separate issue. There are some hacks in the Jsonld extractor I need to fix at the same time the upgrade happens. Better to leave this for ANY23-433.

Reply:
Thanks @HansBrende I am happy to delay this.